meetings/2023-11-08.md

```diff
@@ -0,0 +1,136 @@
+# W3C Solid Community Group: Weekly
+
+* Date: 2023-11-08T14:00:00Z
+* Call: https://meet.jit.si/solid-cg
+* Chat: https://matrix.to/#/#solid_specification:gitter.im
+* Repository: https://github.com/solid/specification
+* Status: Draft
+
+## Present
+* [Sarven Capadisli](https://csarven.ca/#i)
+
+---
+
+## Announcements
+
+### Meeting Guidelines
+* [W3C Solid Community Group Calendar](https://www.w3.org/groups/cg/solid/calendar).
+* [W3C Solid Community Group Meeting Guidelines](https://github.com/w3c-cg/solid/blob/main/meetings/README.md).
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Join queue to talk.
+* Topics can be proposed at the bottom of the agenda to be discussed as time allows. Make it known if a topic is urgent or cannot be postponed.
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/).
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* Aaron Coburn
+
+
+### Introductions
+* Eric Jahn: My main perspective is the need for consent sharing. That would be patient or human services client. 
+* Vivian: I’m interested in interoperability on the web, and I think RDF is the way to go.
```

Comment:
These look like they've already happened, but this meeting doesn't happen for ~21.5 hours (as of this typing)....

I might suggest tweaking this template, so it distinguishes between boilerplate (that doesn't change, and doesn't need to repeat, from meeteing to meeting); structure; actual agenda; and meeting notes/minutes.

It *should* be possible to line each session's agenda up against the minutes for that session, and thereby see what was planned to be discussed Vs. what was actually discussed — most importantly revealing what was *not* discussed, so that cycles onto the next agenda (unless an explicit decision is made to discard some topic(s)).

Reply:
My bad. The meeting hasn't happened. That text is from the previous meeting. I've removed it in https://github.com/solid/specification/pull/595/commits/228c3157de171aae4125374d38a042e8ca526e70